example/src/main/java/com/vimeo/android/deeplink/example/MainActivity.java

```diff
@@ -292,6 +293,17 @@ public void onClick(View v) {
                                 VimeoDeeplink.showAlbums(MainActivity.this, getAlbumsForUserUri());
                             }
                         });
+
+        mIdForAlbumEditText = ((EditText) findViewById(R.id.activity_user_id_album_edit_text));
+
+        configureButton(R.id.activity_main_album_by_id,
```

Comment:
Should there be a radio button for albums as there is for videos, channels, etc? Should it support the ability to enter a album id in the URI Path field?

Reply:
I don't think that it is entirely clear to any outside user what those radio buttons are actually doing which is why I avoided adding to them. The buttons that I added have the same features as might be available in the radio buttons, it is just clearer what is happening. The bottom set of four buttons in the app are the ones I added for the albums feature.